File: 02 - Data Science/06 - CursoPOOUber/Java/UberBlack.java
package Java;

import java.util.ArrayList;
import java.util.Map;

public class UberBlack extends Car {

    Map<String, Map<String,Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial; 

    public UberBlack(String License, Account Driver, Map<String, Map<String,Integer>> typeCarAccepted, ArrayList<String> seatsMaterial){
        super(License, Driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
                
        }
}

File: 02 - Data Science/06 - CursoPOOUber/Python/UberBlack.py
from car import Car

class UberBlack(Car):
    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self, License, Driver, typeCarAccepted, seatMaterial):
        super().__init__(License, Driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatMaterial
        